lib/nyplDataApi.js

```diff
@@ -26,4 +26,8 @@ const bib = (source, id) => {
   return client().get(`bibs/${source}/${id}`)
 }
 
-module.exports = { bib, item }
+const holding = (id) => {
+  return client().get(`holdings?ids=${id}`)[0]
```

Comment:
There are dedicated `GET` endpoints for `Bib` and `Item` records that retrieve a single record. When setting up the `HoldingService` it was decided that we didn't need an endpoint that replicated that functionality. 

Technically we don't, since this should work fine, but it would be a little cleaner if we did. Not sure if it makes sense to add it just for this single use case, but it would make this cleaner and align it with the bib and item methods

Reply:
I agree, in general I think there is value in aligning all of these services, it shouldn't be hard, and it is easy to imagine other services having this need in the future.

@danamansana Could you create a ticket to add this endpoint to the `HoldingService`? We should probably try to get to it this week or next, but not immediately necessary.